ViewControllers/DetailViewController.swift

```diff
@@ -111,7 +123,6 @@ class DetailViewController: UIViewController {
         collectionView.register(DetailCollectionViewCell.self, forCellWithReuseIdentifier: DetailCollectionViewCell.reuseIdentifier)
         collectionView.dataSource = self
         collectionView.delegate = self
-        collectionView.isScrollEnabled = false
```

Comment:
Carefull when adding a scrollview inside a scrollview (in this case we have the scrollview and a collection inside which is also a scrollview). This can lead to some weird funky behaviours. To make things easier for us we usually lock all scrollviews and handle scrolling on one of them.